package.json

```diff
@@ -144,6 +144,7 @@
   },
   "dependencies": {
     "@carbon/utils-position": "1.0.0",
+    "element-closest-polyfill": "^1.0.0",
```

Comment:
This should be in the `devDependencies` and it should be pinned to a specific version (i.e. no `^` or `~`) - `dependencies` are installed whenever the library is installed so we try to keep them to a minimum